.github/pull_request_template.md

```diff
@@ -1,12 +1,14 @@
 <!--
 Thanks a lot for making a contribution to SFML! 🙂
 
-Before you create the pull request, we ask you to check the follow boxes. (For small changes not everything needs to ticked, but the more the better!)
+Please make sure you are targetting the correct branch. No more features are planned for the 2.x branches! (See State of Development in the readme)
```

Comment:
Maybe `readme` could be a link to the readme

Reply:
I'm avoiding making the line too long, so I changed the text and linked to the section